webpack.config.prod.js

```diff
@@ -10,11 +10,11 @@ module.exports = {
   devtool: 'source-map',
 
   entry: {
-    app: ['./client/index.js'],
+    app: [ './client/index.js' ],
   },
 
   output: {
-    path: path.join(__dirname, 'public'),
+    path: path.join(__dirname, 'build'),
```

Comment:
I'm not against renaming the output dir `public/ -> build/`. But so far I haven't seen much consensus on what the alternative to public should be. What do you think about `dist/` instead of `build/`? Anecdotally I feel that I've seen `dist/` more often.